src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        DriverService driverService = (DriverService) injector.getInstance((DriverService.class));
+        Long carId = Long.valueOf(req.getParameter("carId"));
```

Comment:
```suggestion
        Long carId = Long.valueOf(req.getParameter("car_id"));
```
let's use a snake case for the names of the parameters